README.md

```diff
@@ -0,0 +1,188 @@
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [Key Features](#key-features)
+- [🔭 Future Features](#future-features)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Math Magicians] <a name="about-project"></a>
+
+>This project gives you a calculator which is build by react.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Html, Css,Javascript,ReactJs, linter, Git, Github, Node, NPM
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">Html</a></li>
+        <li><a href="#">Css</a></li>
+        <li><a href="#">Javascript</a></li>
+        <li><a href="#">React js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="#">Github</a></li>
+      <li><a href="#">Github Pages</a></li>
+  </ul>
+</details>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+>You need to have Code Editor Installed.
+>Make sure Nodejs and NPM is installed in your system for linters.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```bash
+git clone https://github.com/ghreza-crypto/math-magicians.git
+```
+2. Open terminal and cd into project
+```bash
+cd math-magicians
+```
+3. Run the project with:
+```bash
+npm start
+```
+
+### Install
+
+Install this project with: linters
+
+To check linters run the following command:
+1. Install WebHint:
+```bash
+npm install --save-dev hint@7.x
+```
+2. Install StyleLint
+```bash
+npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x 
+```
+3. Install eslint
+```bash
+npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x 
+```
+## Check Linters Error
+1. To check CSS or SCSS linters error:
+```bash
+npx stylelint "**/*.{css,scss}"
+```
+2. To fix CSS or SCSS linters error:
+```bash
+npx stylelint "**/*.{css,scss}" --fix
+```
+3. To check HTML linters error
+```bash
+npx hint .
+```
+4. To check JavaScript linters error
+```bash
+npx eslint .
+```
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[using its calculator]**
+- **[single page application which makes using easy]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[mobile version functionality]**
```

Comment:
Great job @ghreza-crypto 🤝

- [ ] [Optional] Kindly ensure this feature is relevant to this project. I would suggest you add a feature like `Add a calculator that uses Single Page Application`.